rig-core/src/providers/openrouter/completion.rs

```diff
@@ -292,16 +156,30 @@ impl completion::CompletionModel for CompletionModel {
             "temperature": completion_request.temperature,
```

Comment:
To support tools invocation this should be tools declaration too like `"tools": completion_request.tools,`